cli/args/flags.rs

```diff
@@ -4298,6 +4401,41 @@ mod tests {
     ($($x:expr),* $(,)?) => (vec![$($x.to_string()),*]);
   }
 
+  #[test]
+  fn fast_path() {
+    for args in [
+      svec!["deno"],
+      svec!["deno", "run", "script.js"],
+      svec!["deno", "run", "--allow-all", "script.ts"],
+      svec!["deno", "run", "-A", "script.ts"],
+    ] {
+      assert_eq!(
```

Comment:
Explicit test to ensure that we parse all fast-path commands the same as we'd parse with clap.